### DISCUSSION TITLE
Display Stacking order

### DISCUSSION THREAD

**Discussion by TalentlessGoon** [inkstitch/inkstitch#2626] [Ideas]
Can we have a way of having a numerical or alphabetical display of the stacking order.

![ink](https://github.com/inkstitch/inkstitch/assets/129899482/a8907712-264e-4996-8528-42f666c43c06)

**Comment by kaalleen** [comment]
Interesting idea. Thank you for the feature request.

Moved request over to issues: https://github.com/inkstitch/inkstitch/issues/2651
